pkg/config/flowaggregator/config.go

```diff
@@ -73,3 +75,18 @@ type APIServerConfig struct {
 	// TLS min version.
 	TLSMinVersion string `yaml:"tlsMinVersion,omitempty"`
 }
+
+type ClickHouseConfig struct {
+	// Username is the username for ClickHouse client.
+	Username string `yaml:"username,omitempty"`
+	// Password is the username for ClickHouse client.
+	Password string `yaml:"password,omitempty"`
+	// Database is the name of database where antrea table is created.
+	Database string `yaml:"database,omitempty"`
+	// DbURL is the url to the database. TCP protocol is required.
+	DbURL string `yaml:"dbUrl,omitempty"`
```

Comment:
Nit: maybe DBURL or DatabaseURL matches our naming convention better (abbrs. should be all upper cases).

Reply:
renamed to DatabaseURL